### PR TITLE
✨ caller blame

### PR DESCRIPTION
**Pull request by MatejBabis**

### CODE REVIEW

**Comment by cathackk** [comment]
Thanks for your contribution! Let's just discard the big "blame" banner.

File: loopcoaster/listener.py [hunk 1]
```diff
@@ -49,6 +50,9 @@ def listen(receive_url: str, user: str, password: str, wait_seconds: float, buff
                 ts = datetime.fromtimestamp(obj['ts'])
                 log(f'- [{index}/{len(received)}] {sender}@{ts:%Y-%m-%dT%H:%M:%S}: {message}')
 
+                log(pyfiglet.figlet_format("blame", font="doh", width=100))
+                log(pyfiglet.figlet_format(sender, font="doh", width=100))
```
Comment (cathackk) [inline loopcoaster/listener.py]: Please remove the `blame` line.

Comment (MatejBabis) [reply loopcoaster/listener.py]: Great catch, what a silly mistake!

**Review by cathackk — APPROVED** [review]
Great job! Please just attach a PR description next time.

**Pull request merged**